package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript-ocelot",
-  "version": "0.23.0",
+  "version": "0.23.1",
```

Comment:
Not being able to close modals is arguably a breaking change.

Reply:
Haha my logic was: Any downstream consumers of ocelot's code will still compile, but I see your point. I can do a major version bump if you prefer?

Reply:
Seems more appropriate, but I don't feel strongly enough to block. I think some people outside our organization use Ocelot, though I'd be surprised if it were used by some other company.